config/blast.php

```diff
@@ -43,17 +88,11 @@
         ],
     ],
 
-    'storybook_global_types' => [],
-
-    // set a global custom order for stories in the Storybook UI
-    // More info - https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories
-    'storybook_sort_order' => [],
-
```

Comment:
Sorry, I missed this. It looks like this has been removed in error too. Can you add it back in. Thanks!

Reply:
@mrtimbrook Not sure what happened there as looks like it is there (just checked in the branch and local)

Reply:
@mrtimbrook It's there now please see Commits on Nov 16, 2021 - Update blast.php

Reply:
Oh yeah, I missed it in the diff above 🤦  Thanks!